### ISSUE TITLE
ContextStorage funktioniert nicht

### ISSUE THREAD

**Issue by thmeger** [energychain/node-red-gsi#4]
Willkommen bei Node-RED!
===================
12 Jan 20:18:54 - [info] Node-RED Version: v1.2.2
12 Jan 20:18:54 - [info] Node.js  Version: v10.23.0
12 Jan 20:18:54 - [info] Linux 5.4.72-v7+ arm LE
12 Jan 20:18:55 - [info] Paletten-Nodes werden geladen
12 Jan 20:19:12 - [info] RedBot version: 0.19.7 (node-red-contrib-chatbot)
12 Jan 20:19:13 - [info] Dashboard version 2.26.1 started at /ui
12 Jan 20:19:17 - [info] Einstellungsdatei: /home/pi/.node-red/settings.js
**12 Jan 20:19:17 - [info] Kontextspeicher: 'default' [ module=localfilesystem]**
12 Jan 20:19:17 - [info] Benutzerverzeichnis: /home/pi/.node-red
12 Jan 20:19:17 - [warn] Projekte inaktiviert: editorTheme.projects.enabled=false
12 Jan 20:19:17 - [info] Flow-Datei: /home/pi/.node-red/flows_NodeRedPi.json
12 Jan 20:19:17 - [info] Server wird jetzt auf http://127.0.0.1:1880/ ausgeführt.
12 Jan 20:19:18 - [info] Flows starten
12 Jan 20:19:21 - [info] Flows gestartet
**Corrently GSI requires persistent storage for global values. Consider enable contextStorage in your settings.js
Corrently GSI requires persistent storage for global values. Consider enable contextStorage in your settings.js
Corrently GSI requires persistent storage for global values. Consider enable contextStorage in your settings.js**

Trotz der einkommentierten Einstellungen in der settings.js wird der contextStorage durch GSI nicht genutzt.

**Comment by brau01ma** [comment]
Same on my installation..

**Comment by zoernert** [comment]
@brau01ma :
> Same on my installation..

Which Node JS Version are you on?

**Comment by brau01ma** [comment]
> @brau01ma :
> 
> > Same on my installation..
> 
> Which Node JS Version are you on?

1 Jan 11:41:44 - [info] Node-RED version: v1.2.7
11 Jan 11:41:44 - [info] Node.js version: v12.20.1
11 Jan 11:41:44 - [info] Linux 5.4.79-v7+ arm LE
11 Jan 11:41:45 - [info] Loading palette nodes
11 Jan 11:41:48 - [info] Dashboard version 2.26.1 started at /ui
11 Jan 11:41:48 - [info] Settings file : /home/pi/.node-red/settings.js
11 Jan 11:41:48 - [info] Context store : 'default' [module=localfilesystem]
11 Jan 11:41:48 - [info] User directory : /home/pi/.node-red
11 Jan 11:41:48 - [warn] Projects disabled : editorTheme.projects.enabled=false
11 Jan 11:41:48 - [info] Flows file : /home/pi/.node-red/flows_raspberrypi.json
11 Jan 11:41:49 - [info] Server now running at http://127.0.0.1:1880/
11 Jan 11:41:49 - [warn]

**Comment by zoernert** [comment]
I try to reproduce this error... 

Which Nodes are you using (like Discovergy, Keba, Varta...) ? 

However this is just a warning message if there is no way to determine the zip code (Postleitzahl). In could solved by setting a value in your global data named "zip" and value your location's zip code.

Do you have any futher log messages like:
```
 14 Jan 00:37:18 - [red] Uncaught Exception:
14 Jan 00:37:18 - SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at Request._callback (/mnt/dropbox/Development/corrently/node-red-gsi/node_modules/gsi-discovergy/lib/gsi_discovergy.js:44:31)
```
As this indicates a completely different angle of attack

**Comment by brau01ma** [comment]
Just want to use Node "best hour".

Get this Message:

`Corrently GSI requires persistent storage for global values. Consider enable contextStorage in your settings.js
(node:1746) UnhandledPromiseRejectionWarning: [object Array]
(node:1746) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 28)
(node:1746) UnhandledPromiseRejectionWarning: [object Array]
(node:1746) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 29)
Corrently GSI requires persistent storage for global values. Consider enable contextStorage in your settings.js`

**Comment by zoernert** [comment]
fixed as of v1.5 (latest is v1.9)
